torchtune/modules/transformer.py

```diff
@@ -209,6 +368,10 @@ def forward(
                 before the softmax. A value of True in row i and column j means token i attends
                 to token j. A value of False means token i does not attend to token j. If no
                 mask is specified, a causal mask is used by default. Default is None.
+            encoder_input (Optional[Tensor]): Optional second input to cross attend with x. Shape
+                [batch_size x seq_length x embed_dim] (seq_length and embed_dim may very from x)
+            encoder_mask (Optional[Tensor]):  Cross attention boolean tensor with shape
```

Comment:
Add some information around what the 1s and 0s in the mask mean. We had some users open up issues around this previously and we committed to describing this in more detail